apps/emqx/rebar.config

```diff
@@ -24,7 +24,7 @@
 {deps, [
     {emqx_utils, {path, "../emqx_utils"}},
     {lc, {git, "https://github.com/emqx/lc.git", {tag, "0.3.2"}}},
-    {gproc, {git, "https://github.com/uwiger/gproc", {tag, "0.8.0"}}},
+    {gproc, {git, "https://github.com/emqx/gproc", {tag, "0.9.1-emqx0"}}},
```

Comment:
is the vsn in app.src `0.9.1`, or with a suffix ?

Reply:
nah, it's using `git`. we'll need to use a better app vsn for it.

Reply:
The problem here is that upstream has not tagged that version yet.  So, if we tag `0.9.1` in our fork, it might be strange when upstream does so.  So I used that suffix to differentiate it.